common.js

```diff
@@ -0,0 +1,37 @@
+// Verify user's code, returns boolean.
```

Comment:
This doesn't seem done; should we leave it for another PR?

Reply:
No, just was using this as example code for refactor. I'll fill these in in new PR.